File: python/URI/1234.py
while True:
    try:
        dance = input()
        new = ""
        mode = True
        for l in dance:
            if l.isdigit() or l.isspace():
                new += l
            elif (mode == True and l.isalpha()):
                new += l.upper()
                mode = False
            elif (mode == False and l.isalpha()):
                new += l.lower()
                mode = True
        print(new)
    except:
        break

File: C/URI/1234.c
#include <stdio.h>
#include <ctype.h>
#include <string.h>

void main()
{
    char dance[50];
    int mode, i;
    while (1)
    {
        fpurge(stdin);
        gets(dance);
        if (feof(stdin))
            break;
        mode = 0;
        i = 0;
        for (i = 0; i < strlen(dance); i++)
        {
            if (mode == 0 && isalpha(dance[i]))
            {
                dance[i] = toupper(dance[i]);
                mode = 1;
            }
            else if (mode == 1 && isalpha(dance[i]))
            {
                dance[i] = tolower(dance[i]);
                mode = 0;
            }
            printf("Modo: %i\n", mode);
        }
        printf("%s\n", dance);
    }
}